src/quo/previews/lists.cljs

```diff
@@ -0,0 +1,110 @@
+(ns quo.previews.lists
+  (:require [reagent.core :as reagent]
+            [quo.core :as quo]
+            [quo.react-native :as rn]
+            [quo.design-system.colors :as colors]
+            [quo.previews.preview :as preview]))
+
+(def all-props (preview/list-comp [] {}))
+
+(defn avatar []
+  [rn/view {:border-radius    20
```

Comment:
probably  it would be better to avoid confusing move all components to quo, even view